pkg/operator/ceph/object/user.go

```diff
@@ -171,11 +171,12 @@ func DeleteUser(c *Context, id string, opts ...string) (string, int, error) {
 	}
 	result, err := runAdminCommand(c, args...)
 	if err != nil {
+		if strings.Contains(result, "user does not exist") {
+			return "", RGWErrorNotFound, errors.Wrapf(err, "user %q does not exist so cannot delete", id)
```

Comment:
If the user doesn't exist, shouldn't we return success instead of an error?

Reply:
True, this means a bit more refactor though...

Reply:
I see now, the caller will look at RGWErrorNotFound